docs/tutorial/avax-nft.md

```diff
@@ -0,0 +1,399 @@
+
+Create a directory named `/contracts` within the project root. Create a smart contract file named `Filet.sol` with the following code:
+
+```js
+// contracts/Filet.sol
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+
+contract Filet is ERC721URIStorage {
+    using Counters for Counters.Counter;
+    Counters.Counter private _tokenIds;
+
+    constructor() ERC721("Filet", "FIT") {}
+
+    function mintTo(address player, string memory tokenURI)
+        public
+        returns (uint256)
+    {
+        _tokenIds.increment();
+        uint256 newItemId = _tokenIds.current();
+        _mint(player, newItemId);
+        _setTokenURI(newItemId, tokenURI);
+
+        return newItemId;
+    }
+}
+
+```
+
+You can come up with your own contract's name, token's name, and token's ticker. I'm naming mine `Filet` here.
+
+Now compile the contract `Filet.sol` using this hardhat command:
+
+```shell
+npx hardhat compile
+> Compiling 1 file with 0.8.0
+> Compilation finished successfully
+```
+
+Once the contract is compiled, create another directory named `script` and add `deploy.js` with the following code:
+
+```js
+import {
+  Contract,
+  ContractFactory
+} from "ethers"
+import { ethers } from "hardhat"
+
+const deploy = async (contractName) => {
+  const Contract = await ethers.getContractFactory(contractName)
+  const contract = await Contract.deploy()
+
+  await contract.deployed()
+  console.log(`${contractName} deployed to: ${contract.address}`)
+}
+
+const main = async () => {
+  await deploy("Filet")
+}
+
+main()
+.then(() => process.exit(0))
+.catch(error => {
+  console.error(error)
+  process.exit(1)
+})
+```
+
+This script uses [Ethers](https://docs.ethers.io/v5/) library to deploy the contract to the local Avalanche node(s). Now, deploy the contract with:
+
+```shell
+npx hardhat run scripts/deploy.js --network local
+> Filet deployed to: 0x5FbDB2315678afecb367f032d93F642f64180aa3
+```
+
+We now have a token contrat deployed at `0x5FbDB2315678afecb367f032d93F642f64180aa3`.
+
+Note that up until now, the steps are the same for both Ethereum and Avalanche.
+
+Now let's spin up Hardhat's [developer console](https://hardhat.org/guides/hardhat-console.html) to start interacting with our `Filet` contract:
+
+```shell
+npx hardhat console --network local
+> Welcome to Node.js v14.18.1.
+> Type ".help" for more information.
+> >
+```
+
+Now, type the following into the prompt to instantialize the contract object:
+
+```js
+>> const Filet = await ethers.getContractFactory("Filet")
+>> const filet = await Filet.attach("0x5FbDB2315678afecb367f032d93F642f64180aa3")
+```
+
+Next, we inspect the accounts and the balances:
```

Comment:
maybe more specific on inspecting balances of the new created NFT token. Otherwise, it might be confusing for ETH or AVAX.